src/samples/wallet_demo.ts

```diff
@@ -52,9 +52,11 @@ doc.getElementById('publishAddresses').innerHTML = Object.keys(sampleAddressMap)
 
 // --- @crux/js-sdk integration --- //
 // defining cruxClientOptions
-const cruxClientOptions: ICruxPayPeerOptions = {
+const cruxClientOptions: ICruxClientOptions = {
     getEncryptionKey: () => encryptionKey,
```

Comment:
Can you remove getEncyrptionKey from the walletDemo? Lets keep it minimal so that wallets can refer to this code to integrate.